enterprise-wifi.html

```diff
@@ -0,0 +1,126 @@
+        <link type="text/css" rel="stylesheet" href="/css/style.css" />
+        <meta name="keywords" content="download,server" />
+        <!--[if lt IE 7.]>
+        <script defer type="text/javascript" src="/pngfix.js"></script>
+        <![endif]-->
+    </head>
+    <body>
+        <!--#include file="/header.html" -->
+        <div id="content">
+            <h1>Enterprise Wi-Fi / IEEE 802.1X</h1>
+            <h2>Deficiencies of PSK networks</h2>
+            <p>Common home-use Wi-Fi networks do not need a RADIUS server because
+                they “secure” the network with <strong>one single network key</strong>, the 
+                “WPA/WPA2 Pre-Shared Key” (PSK). That key is the same for every user, 
+                is often guessable, and <strong>can’t be revoked for one user</strong> (if one user should be 
+                denied access, the key needs to be changed for the entire network
+                and every user needs to be given the new key). When a network is
+                sniffed, an attacker can perform <strong>offline attacks</strong> to guess the key.</p>
+            <p><strong>All this makes PSK networks unfit for enterprise use.</strong></p>
+            <h2>IEEE 802.1X and RADIUS Authentication</h2>
+            <p>The IEEE standards for Wi-Fi (IEEE 802.11) foresee an “Enterprise”
+                mode which is fundamentally different from PSK networks because
+                the Wi-Fi encryption keys are provisioned <strong>per user</strong>
+                and <strong>per session</strong>. Every user needs to authenticate
+                with his <strong>personal credentials</strong>; at that moment a 
+                key is generated and is communicated to the user’s device and the
+                NAS he connects to.</p>
+            <p>Before sending his authentication credentials,
+                the <strong>network must authenticate to the user</strong>, proving
+                that it is indeed genuine; only then is the client’s credential
+                released. The IEEE standard IEEE 802.1X (using RADIUS and the
+                Extensible Authentication Protocol, EAP) is used for authentication
+                and key management.</p>
+            <p>Enterprise Wi-Fi authentication also enables advanced features
+                such as putting users dynamically into a specific VLAN (e.g. 
+                separate “guest” and “staff” logins into different IP networks
+                even though being on the same SSID).</p>
+            <p>Enterprise Wi-Fi requires</p>
+            <ol>
+                <li>a RADIUS server which can do EAP authentication</li>
+                <li>Wi-Fi equipment which is correctly configured to use RADIUS authentication</li>
+                <li>user devices configured to do Enterprise Wi-Fi correctly</li>
+            </ol>
+            <h2>RADIUS Server</h2>
+            (FreeRADIUS advertising :-) )
```

Comment:
See my original pull request - I simply need some directions what kind of text should be in there:

"The RADIUS server section was "intentionally left blank". Since the
page goes onto freeradius.org, folks should already be clear which
RADIUS server to use, but if you want to stress how great exactly
FreeRADIUS is, or link to the Wiki's EAP or VLAN attributes pages - do
whatever you want there."

I'm perfectly happy to ghostwrite something if you don't have stock text about how great FreeRADIUS is? Just let me know...